src/ComplianceChecker.ts

```diff
@@ -35,8 +36,10 @@ export class ComplianceChecker {
 
   async main() {
     const reporters: Reporter[] = [new ConsoleReporter()]
-    const reporter = new MultiReporter(reporters)
+    if (CloudNotifierReporter.enabled())   
+        reporters.push(new CloudNotifierReporter())
```

Comment:
<details>
<summary>:warning: <strong>This code contains single line control structures without curly braces, `{}`.</strong> (click for details)</summary>

Consider surrounding these statements with `{ }`.  
  
Braces are used around all statements, even singletons, when they are part of a control structure, such as an `if`-`else` or `for` statement.  
  
This makes it easier to add statements without accidentally introducing bugs due to forgetting to add braces. Readability is also significantly improved.

**[More details](https://confluence.devfactory.com/x/fHvlFQ)**
</details>

[//]: # (<a id="f44cdbcd-e4ce-4ee1-8508-71cc85a02f0f" data-finding-id="367360892" href="none" codereview-uuid-reserved-tag data-business-id="932b6095-04c8-4079-a6b1-1d6d1eb22183" />)

Reply:
<details>
<summary>:warning: <strong>Formatting related to code indentation is inconsistent.</strong> (click for details)</summary>

A possible improvement would be to apply the current language Coding Conventions (or project-specific style) to all files and fix any possible indentation issue before committing to maintain code style consistency.  
  
This change would ensure that the code is consistently indented, thus, improving readability, making variable and block scopes easier to understand.

**[More details](https://confluence.devfactory.com/x/cnvlFQ)**

## The improvement suggestion in this case
It's recommended 2 spaces for indentation but even if `2 spaces for indentation` rule is not being followed in the project, consider using the same indentation character(s) in all lines.  
```suggestion  
    if (CloudNotifierReporter.enabled()) {
      reporters.push(new CloudNotifierReporter())
    }
```
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence(s).
</details>

[//]: # (<a id="42af1101-a82f-4af9-9321-ff230c29cdc7" data-finding-id="367360882" href="none" codereview-uuid-reserved-tag data-business-id="932b6095-04c8-4079-a6b1-1d6d1eb22183" />)